client/rest/src/routes/networkRoutes.js

```diff
@@ -87,6 +107,47 @@ module.exports = {
 				next();
 			}));
 
+		server.get('/network/inflation', (req, res, next) => readAndParseInflationPropertiesFile()
+			.then(inflationInflectionPoints => {
+				res.send(inflationInflectionPoints.map(point => ({
+					// send BigInts over network as strings
+					startHeight: point.startHeight.toString(),
+					rewardAmount: point.rewardAmount.toString()
+				})));
+				next();
+			}).catch(() => {
+				res.send(errors.createInvalidArgumentError('there was an error reading the inflation properties file'));
+				next();
+			}));
+
+		server.get('/network/inflation/at/:height', (req, res, next) => readAndParseInflationPropertiesFile()
+			.then(inflationInflectionPoints => {
+				const height = BigInt(req.params.height);
+
+				const findMatchingPoint = () => {
+					const firstPoint = inflationInflectionPoints[0];
+					if (height < firstPoint.startHeight)
+						return { startHeight: 'N/A', rewardAmount: '0' };
```

Comment:
isn't nemesis block falling into this branch?

Reply:
yes, and, imo, it should as it has 0 block rewards